src/handformer.cpp

```diff
@@ -0,0 +1,74 @@
+#include "handformer.h"
+
+#include <ctype.h>
+
+#include <algorithm>
+#include <cstdint>
+#include <iterator>
+#include <map>
+#include <string>
+#include <vector>
+
+#include "piecetype.h"
+
+using Mahjong::Piece;
+
+std::map<char, Piece::Type> NOTATION_TO_SUIT = {
+  {'m', Piece::CHARACTER_SUIT},
+  {'s', Piece::BAMBOO_SUIT},
+  {'p', Piece::PIN_SUIT},
+  {'z', Piece::HONOR_SUIT},
+};
+
+std::map<uint8_t, Piece::Type> NOTATION_TO_HONOR = {
+  {1, Piece::EAST_WIND},
+  {2, Piece::SOUTH_WIND},
+  {3, Piece::WEST_WIND},
+  {4, Piece::NORTH_WIND},
+  {5, Piece::WHITE_DRAGON},
+  {6, Piece::GREEN_DRAGON},
+  {7, Piece::RED_DRAGON},
+};
```

Comment:
Please make const

Reply:
Fixed in https://github.com/realliance/libmahjong/commit/5241a20f2ebc3016988be2085fe02acfe2b1c516